### ISSUE TITLE
PhoneGap example directory from github stopped working February 4

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#14]
```
What steps will reproduce the problem?
1. Download phonegap-android from github
2. Build phonegap example project (without JQM or Sencha)
3. Deploy and run
4. Only the dialer will work

On February 4, example/index.html changed from including phonegap.js to 
including phonegap.0.9.4.min.js. A workaround is to edit index.html back to 
phonegap.js

The bug fix should build phonegap.0.9.4.min.js in Eclipse


```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 16 Feb 2011 at 3:00

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 16 Feb 2011 at 3:01
- Added labels: **Milestone-Release1.2.0**

**Comment by GoogleCodeExporter** [comment]
```
For 1.2, the fix wil be to insure index.html refers to phonegap.js.  
Minification will be done for a future release. It's tracked now in Issue 11
```

Original comment by `paul.beu...@gmail.com` on 21 Feb 2011 at 11:27
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 1.2 release
```

Original comment by `paul.beu...@gmail.com` on 23 Feb 2011 at 4:24
- Changed state: **Fixed**
